atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/WorkloadServerLauncher.java

```diff
@@ -0,0 +1,128 @@
+import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+import io.dropwizard.Application;
+import io.dropwizard.jackson.DiscoverableSubtypeResolver;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+
+public class WorkloadServerLauncher extends Application<WorkloadServerConfiguration> {
+
+    private static final SafeLogger log = SafeLoggerFactory.get(WorkloadServerLauncher.class);
+    private static final UserAgent USER_AGENT = UserAgent.of(Agent.of("AtlasDbWorkloadServer", "0.0.0"));
+
+    private final TaggedMetricRegistry taggedMetricRegistry = new DefaultTaggedMetricRegistry();
+    private final CountDownLatch workflowsRanLatch = new CountDownLatch(1);
+
+    public static void main(String[] args) throws Exception {
+        new WorkloadServerLauncher().run(args);
+    }
+
+    @Override
+    public void initialize(Bootstrap<WorkloadServerConfiguration> bootstrap) {
+        MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate("AtlasDbWorkLoadServer");
+        bootstrap.setMetricRegistry(metricRegistry);
+        bootstrap.setObjectMapper(ObjectMappers.newServerObjectMapper());
+        bootstrap.getObjectMapper().setSubtypeResolver(new DiscoverableSubtypeResolver());
+        bootstrap.getObjectMapper().registerModule(new Jdk8Module());
+        super.initialize(bootstrap);
+    }
+
+    @Override
+    public void run(WorkloadServerConfiguration configuration, Environment environment) {
+        environment.getObjectMapper().registerModule(new Jdk8Module()).registerModule(new JavaTimeModule());
+
+        ExecutorService workflowRunnerExecutor =
+                environment.lifecycle().executorService("workflow-runner").build();
+
+        workflowRunnerExecutor.execute(() -> runWorkflows(configuration, environment));
+    }
+
+    private void runWorkflows(WorkloadServerConfiguration configuration, Environment environment) {
+        ExecutorService singleRowTwoCellsExecutorService = environment
+                .lifecycle()
+                .executorService(SingleRowTwoCellsWorkflows.class.getSimpleName())
+                .build();
+        MetricsManager metricsManager = MetricsManagers.of(environment.metrics(), taggedMetricRegistry);
+        AtlasDbTransactionStoreFactory transactionStoreFactory = AtlasDbTransactionStoreFactory.createFromConfig(
+                configuration.install().atlas(),
+                Refreshable.only(configuration.runtime().atlas()),
+                USER_AGENT,
+                metricsManager);
+        SingleRowTwoCellsWorkflowConfiguration workflowConfig =
+                configuration.install().singleRowTwoCellsConfig();
+
+        log.info("antithesis: start_faults");
```

Comment:
Ah nice. I'm guessing you got this from their docs?

Reply:
Yes, and over private communication!